tests/cases/fourslash/completionConstructorKeywordAfterPropertyDeclaration.ts

```diff
@@ -0,0 +1,103 @@
+/// <reference path="fourslash.ts" />
+
+//// // situations that `constructor` is partly present
+//// class A {
+////   blah; con/*1*/
+//// }
+//// class B {
+////   blah
+////   con/*2*/
+//// }
+//// class C {
+////   blah;
+////   con/*3*/
+//// }
+//// class D {
+////   blah: number
+////   con/*4*/
+//// }
+//// class E {
+////   blah: number; con/*5*/
+//// }
+//// class F {
+////   blah = 123
+////   con/*6*/
+//// }
+//// class G {
+////   blah = 123; con/*7*/
+//// }
+//// // situations that `constructor` is fully present
```

Comment:
All of these are fine to have, and correct, but as far as I understand, essentially pointless. I don’t think they’re particularly important to test, and I definitely think that little to no implementation code should be written to make these cases pass.

Reply:
Kindly refer to https://github.com/microsoft/TypeScript/pull/43654#discussion_r619398932 maybe.